### DISCUSSION TITLE
3.7.0 to 3.7.2 -> Unable to open app (Mac)

### DISCUSSION THREAD

**Discussion by SJW999** [sn4k3/UVtools#589] [Q&A] [answered]
I have been using 3.7.0 without issues on an M1 MacBook Pro. I downloaded and installed 3.7.2 and the app will not open (I took care of the usual "unidentified developer" issue in Security Preferences). I get a dialog box telling me the app cannot be opened. I deleted the app, downloaded 3.7.0 and now even that version will not open.
Is there a preferences file or library that needs to be deleted? I'll do some searching now but wondered if anyone else ha run into the same problem.
Thanks in advance.

Steve W

**Comment by sn4k3** [comment, marked as answer]
UVtools does not change your system in anyway, maybe you downloaded the arm64 package when you need the x64 instead?

**Reply by SJW999** [reply]
Hmm, I assumed I needed arm64 for my M1 Mac. I will try the x64 version now.

Thanks

**Reply by SJW999** [reply]
Sure enough, x64 runs fine. Sorry for my CPU architecture ignorance.

Thanks

**Reply by sn4k3** [reply]
> Hmm, I assumed I needed arm64 for my M1 Mac.

And you are right, but read the scope of the problem here: https://github.com/sn4k3/UVtools/discussions/440

**Reply by SJW999** [reply]
Now I understand. Thank you very much
